corehq/motech/repeaters/templates/repeaters/repeat_record_report.html

```diff
@@ -11,9 +11,58 @@
         <script src="{% static 'ace-builds/src-min-noconflict/mode-xml.js' %}"></script>
         <script src="{% static 'ace-builds/src-min-noconflict/ext-searchbox.js' %}"></script>
         <script src="{% static 'repeaters/js/repeat_record_report.js' %}"></script>
+        <script src="{% static 'repeaters/js/repeat_record_report_selects.js' %}"></script>
     {% endcompress %}
 {% endblock js %}
 
+{% block filter_panel %} {{ block.super }}
+  <div>
+      {% if total > 0 %}
+        <div id ="form_options" class="well form-inline" style="margin: 1em">
+          <div>
+            <button id="resendAll" class="btn btn-default">Resend selected</button>
+            <button id="cancelAll" class="btn btn-default">Cancel selected</button>
+            <button id="requeueAll" class="btn btn-default">Requeue selected</button>
+            <label class="checkbox" >
+              <input type="checkbox" id="select_all" value="{{ form_query_string|urlencode }}"
+                     onclick="selectAll()" style="margin-left: 1em;">
+                {% blocktrans %}Select all {{ total }} payloads{% endblocktrans %}
+            </label>
+            <label class="checkbox" >
+              <input type="checkbox" id="cancel_all" value="{{ form_query_string_cancellable|urlencode }}"
+                     onclick="selectCallable()" style="margin-left: 1em;">
+                {% blocktrans %}Select {{ total_cancel }} cancellable payloads{% endblocktrans %}
+            </label>
+            <label class="checkbox" >
+              <input type="checkbox" id="requeue_all" value="{{ form_query_string_requeue|urlencode }}"
+                     onclick="selectRequeue()" style="margin-left: 1em;">
+                {% blocktrans %}Select {{ total_requeue }} requeue payloads{% endblocktrans %}
+            </label>
+          </div>
+        </div>
+      {% endif %}
+  </div>
+  <div id="warning" class="alert alert-warning" style="display: none;">
```

Comment:
Nitpick: I'd use `class="hide"` instead of `style="display: none"` - we take a fairly hard line against inline styles.